pyquil/api/_quantum_computer.py

```diff
@@ -654,7 +654,13 @@ def get_qc(name: str, *, as_qvm: bool = None, noisy: bool = None,
 
 
 @contextmanager
-def local_qvm() -> Iterator[Tuple[subprocess.Popen, subprocess.Popen]]:
+def local_runtime(
+        host: str = '127.0.0.1',
```

Comment:
these parameters feel like they should be [keyword-only](https://www.python.org/dev/peps/pep-3102/)

Reply:
I'm unsure how best to do this. Do we list every command line option? just a subset? how do we decide on the subset?

Reply:
Parameters are now keyword-only 👍 
I can add all options which would be useful changing (e.g. not expose things like `--verbose` or `log-level`, but all other runtime options) if you would want that, but that may add additional work if options change (especially making sure it's backwards compatible could be difficult)